Godeps/Godeps.json

```diff
@@ -591,7 +607,7 @@
 		},
 		{
 			"ImportPath": "github.com/golang/protobuf/proto",
-			"Rev": "b982704f8bb716bb608144408cff30e15fbde841"
+			"Rev": "8d92cf5fc15a4382f8964b08e1f42a75c0591aa3"
```

Comment:
> distribution does not use it.

@soltysh But it is listed in [Godeps](https://github.com/docker/distribution/blob/master/Godeps/Godeps.json#L196).